### PR TITLE
docs: initialize FAQs page with Kubernetes unknown args error

### LINKED ISSUE #6378: Add known issues page to docs
As discussed in standup, we should set up a known issues page to mentioned common issues and make them searchable in the docs.

e.g. good candidate to start with is to document k8s / helm behavior in combination with Lodestar env vars
- https://github.com/ChainSafe/lodestar/issues/6045

### PR DESCRIPTION
**Pull request by philknows**
**Motivation**

Closes #6378 

The issue as discussed in [Discord](https://discord.com/channels/593655374469660673/1201684723190210560/1201772954313306112), sparked an idea for us to include a page in our documentation with common questions and known errors that are useful and likely asked by other users. 

**Description**

This PR initializes a FAQ page with a format displayed below using collapsable admonitions for a cleaner visual. This includes our first note to help Kubernetes users deal with env errors.

Screenshot 2024-02-09 at 6 42 28 PM

### CODE REVIEW

**Review by nflaig — CHANGES_REQUESTED** [review]
This does not explain what the root cause is and how to resolve the issue, will push some updates to this shortly

**Comment by nflaig** [comment]
Pushed a few updates to better explain root cause and resolution of the issue

image

**Review by nflaig — DISMISSED** [review]
Looks good to me now, let's get some more feedback. I really like the layout as it is quite minimal and will also allow us to collapse the sections by default in the future if needed.

**Review by nflaig — DISMISSED** [review]

**Review by wemeetagain — DISMISSED** [review]

**Review by nflaig — DISMISSED** [review]

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -42,6 +42,7 @@ markdown_extensions:
   - codehilite:
       guess_lang: false
   - admonition
+  - pymdownx.details
```
Comment (matthewkeil) [inline docs/mkdocs.yml:45]: this is a great add!!!  makes it look really nice

**Review by matthewkeil — APPROVED** [review]
LGTM!! 🚀   

We definitely should use the details module in some other locations

**Review by nflaig — APPROVED** [review]

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.16.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.16.0) :tada:

**Pull request merged**